towhee/engine/operator_pool.py

```diff
@@ -92,7 +94,7 @@ def acquire_op(self, hub_op_id: str,
                 self._all_ops[op_key] = storage
 
             if not storage.op_available():
-                op = self._op_loader.load_operator(hub_op_id, op_args)
+                op = self._op_loader.load_operator(hub_op_id, op_args, framework)
```

Comment:
`Operator Loader` doesn't care about the framework info, we may add framework into op_args.